test/client_test.cc

```diff
@@ -42,6 +42,29 @@ TEST_F(ClientTest, AutoConcurrencyRun) {
   EXPECT_TRUE(program.run());
 }
 
+// TODO(oschaaf): this is just for coverage, and we don't care where
+// any traffic we send ends it or what that looks like.
+// We do functional testing in python now, but unfortunately any code we hit
+// there isn't counted as code-coverage.
+// Ideally, the code hit during the python test runs would count for covergage,
+// and we use unit-tests here to hit any edge cases we can't easily hit otherwise.
+// XXX(oschaaf): look into ways to accomplish that.
```

Comment:
Do you want an issue?  I like the style where you do TODO(issue URI).  It makes it easy to cross-reference the issue and the code and it doesn't tie the work to one person.